### PR TITLE
Add o-brand as a dependency if brand is selected in the prompts

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: templates/bower-json.js [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
+		"node_modules",
+		"bower_components",
+		"build"
+	],
+	"main": [
+		${answers.scss ? `"main.scss"` : ''}${answers.scss && answers.javascript ? "," : ''}
+		${answers.javascript ? `"main.js"` : ''}
+	],
+	"dependencies": {
+		${answers.brand ? `"o-brand": "*"` : ''}
```
Comment (notlee) [inline templates/bower-json.js:15]: Some brand functions are use in the boilerplate, so maybe we should put a version here?

Comment (JakeChampion) [reply templates/bower-json.js:15]:
```suggestion
		${answers.brand ? `"o-brand": "^3.2.10"` : ''}
```

File: templates/bower-json.js [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
+		"node_modules",
+		"bower_components",
+		"build"
+	],
+	"main": [
+		${answers.scss ? `"main.scss"` : ''}${answers.scss && answers.javascript ? "," : ''}
+		${answers.javascript ? `"main.js"` : ''}
+	],
+	"dependencies": {
+		${answers.brand ? `"o-brand": "^3.2.10"` : ''}
```
Comment (JakeChampion) [inline templates/bower-json.js:15]:
```suggestion
		${answers.brands ? `"o-brand": "^3.2.10"` : ''}
```

File: templates/main-scss.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ module.exports = answers => {
 	const name = answers.name;
 
 	return `@import 'src/scss/variables';
+${answers.brand ? `@import 'src/scss/_brand.scss';` : ''}
```
Comment (JakeChampion) [inline templates/main-scss.js:7]:
```suggestion
${answers.brands ? `@import 'src/scss/_brand.scss';` : ''}
```

**Comment by JakeChampion** [comment]
:tada: This PR is included in version v0.2.0 :tada:

**Pull request merged**